lib/chef/http/ssl_policies.rb

```diff
@@ -102,7 +103,11 @@ def set_client_credentials
       end
 
       def config
```

Comment:
yeah we should have standardized mixins for `def config`, `def env`, and the typical REST object API at a minimum.